src/ui/common/form/RenderSwitchInput.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Switch } from 'patternfly-react';
+
+const TRUE_VALUE = 'true';
+
+const RenderSwitchInput = ({ input }) => {
```

Comment:
Duplicated of SwitchRenderer

Reply:
It seems SwitchRenderer does not work properly with input values.